### ISSUE TITLE
 Local resources/images are not always properly resolved

### ISSUE THREAD

**Issue by gernotstarke** [aim42/htmlSanityCheck#210]
and its specifications

**Comment by thc202** [comment]
May I suggest changing the issue title to `Local resources/images are not always properly resolved`? Suggesting that because that's the main issue regarding the dir paths, both checkers (`MissingLocalResourcesCheck` and `MissingImageFilesChecker`) might not always resolve the links correctly, for example, all the following files exist yet they are reported as missing:
```
Missing Local Images Check
2 img src attributes checked, 2 missing image files found.

    image "../images/16/101grey.png" missing
    image "../images/16/101.png" missing
```

```
Missing Local Resources Check
4 anchor tag href attribute checked, 4 missing local resources found.

    local resource "../../../start/concepts/scope.html" missing
    local resource "../../../start/concepts/pscan.html" missing
    local resource "../../overview.html" missing
    local resource "options.html" missing
```

I already have a fix locally but still working in progress, I will push it sooner to gather feedback though.

**Comment by gernotstarke** [comment]
thanx, you're right.
btw - I'm Gernot from Cologne, do you mind telling me who you are?

**Comment by thc202** [comment]
Pull request opened: #213, let me know what you think.

**Comment by gernotstarke** [comment]
just tested (successfully) and merged - thanx!
